app/views/culinary_recipes/show.html.erb

```diff
@@ -0,0 +1,48 @@
+<div class="mx-auto md:w-2/3 w-full flex">
+  <%= form_with(model: @culinary_recipe, local: true) do |form| %>
+  <div class="flex items-center justify-center">
+    <%= form.check_box :public, class: "toggle-checkbox", id: "public-toggle" %>
+    <%= form.label :public, "Public", class: "toggle-label" %>
+  </div>
+<% end %>
+  <div class="mx-auto w-full">
+
+    <%= render @culinary_recipe %>
+
+    <%= link_to "Back to culinary_recipes", culinary_recipes_path, class: "ml-2 mt-10 rounded-lg py-3 px-5 bg-gray-100 inline-block font-medium" %>
+
+    <%= link_to 'Generate Shopping List', method: :post, class: "l-2 mt-10 rounded-lg py-3 px-5 bg-gray-100 inline-block font-medium" %>
+
```

Comment:
- You have done well so far, but when I click on the link to generate a shopping list, nothing happens, please fix this issue and ensure it is possible to generate a shopping list according to the project requirements 👍 👍

Reply:
Hello @zieeco , @mckent05 , guys I think there is something wrong I did not submit the request since I am working on the first required changes, could you text me on slack

Reply:
- [ ] [REQUIRED] As stated by the previous reviewer, I am also unable to generate the shopping list according to the project requirement. Kindly fix this.